torch/quantization/quantize.py

```diff
@@ -137,14 +136,18 @@ def insert_activation_post_process(m, special_act_post_process=None):
             m._forward_hooks.move_to_end(handle.id, last=False)
 
     for name, child in module.named_children():
```

Comment:
would it make sense to add a stronger test for this section to help catch future issues?  For example, something like a model which has modules going through all of the branches in these if statements, passing some data through it, and verifying that every observer has seen values?

Reply:
I think we do have tests for each of the branches, although it's probably spread out in different op tests.

Reply:
if we make an artificial model, not sure it would be runnable or not actually.

Reply:
it doesn't have to be this PR (since we are adding an fb-only test), but seems like just increasing integration model coverage for matching numerics between eager and FX would be valuable.  It seems like `TestQuantizeFxModels` has various models commented out, perhaps we can invest in increasing coverage a bit.

Reply:
yeah I think we should add more e2e tests as well